doc/code-submission-guidelines.md

```diff
@@ -23,30 +23,21 @@ Each commit message should consist of a header, a body and a footer. The header
 that includes a type, a scope and a subject. Here's a sample of the format:
 
 ```
-<type>
-  (<scope>):
-    <subject>
-      <BLANK LINE>
-      <body>
-        <BLANK LINE>
-        <footer>
-        </footer>
-        </BLANK>
-      </body>
-      </BLANK>
-    </subject>
-  </scope>
-</type>
+<type>(<scope>): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
 ```
 
 ### Here's an example:
 
 ```sh
 perf(rule): improve speed of color contrast rules
 
-Use async process to compare elements without UI lockup
+  Use async process to compare elements without UI lockup
```

Comment:
This should not be indented

Reply:
Ha ha, I know what you mean, but changing our docs is outside scope of this PR. I will bring this is up as a doc change and do the needy.

Reply:
That's a matter of style, Git can handle indented commit messages just fine. But we haven't followed that in practice, so it's probably safe to remove.

Reply:
Thanks @marcysutton